mmdet/datasets/samplers/class_aware_sampler.py

```diff
@@ -0,0 +1,175 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import math
+
+import torch
+from mmcv.runner import get_dist_info
+from torch.utils.data import Sampler
+
+
+class ClassAwareSampler(Sampler):
+    r"""A class-aware sampling strategy to effectively tackle the
```

Comment:
The summary should be short and precise.

Also describe whether it can be used in distributed or non-distributed scenarios.

Reply:
Seems this sampler can be used in both dist and non-dist scenarios